dll/win32/mshtml/olecmd.c

```diff
@@ -241,6 +241,11 @@ static HRESULT exec_print(HTMLDocument *This, DWORD nCmdexecopt, VARIANT *pvaIn,
         return S_OK;
     }
 
+    // returning here fixes CORE-16884. Maybe use this until printing works.
+    ERR("Sorry, ReactOS does not support printing yet.\n");
+    return S_OK;
+
+    // For now if this is executed we get internal corruptions for unknown reasons. 
```

Comment:
This file belongs to Wine code base, please use `__REACTOS__` guards.

Reply:
should be resolved now